### PR TITLE
meshregistry: allows ns and group info to be injected into the meta of nacos instance

### PR DESCRIPTION
**Pull request by believening**

### CODE REVIEW

File: staging/src/slime.io/slime/modules/meshregistry/pkg/source/nacos/httpclient.go [hunk 1]
```diff
@@ -19,6 +21,20 @@ const (
 
 var (
 	serviceListPageSize = 1000
+
+	injectNsGroupIntoMeta, _ = strconv.ParseBool(os.Getenv("NACOS_META_INJECT_NSGROUP"))
```
Comment (YonkaFang) [inline staging/src/slime.io/slime/modules/meshregistry/pkg/source/nacos/httpclient.go:25]: use module config instead of env

Comment (believening) [reply staging/src/slime.io/slime/modules/meshregistry/pkg/source/nacos/httpclient.go]: done

File: staging/src/slime.io/slime/modules/meshregistry/pkg/bootstrap/args.go [hunk 2]
```diff
@@ -171,6 +171,9 @@ type NacosSourceArgs struct {
 	Password string
 	// fetch services from all namespaces, only support Polling mode
 	AllNamespaces bool
+	// If enabled, the namespace and group information will be injected into ep's metadata.
+	// Use env NACOS_NAMESPACE_META_KEY and NACOS_GROUP_META_KEY to configure the keys to be injected.
+	InjectNsGroupIntoMeta bool
```
Comment (YonkaFang) [inline staging/src/slime.io/slime/modules/meshregistry/pkg/bootstrap/args.go:176]: The combined use of env and module configuration is too heavy-handed for this simple scenario.

`MetaKeyGroup, MetaKeyNamespace string`

```go
if xx.MetaKeyGroup != "" {
  inst.Metadata[xx.MetaKeyGroup] = namespaceId
}
```

Comment (believening) [reply staging/src/slime.io/slime/modules/meshregistry/pkg/bootstrap/args.go]: replaced `MetaKeyGroup, MetaKeyNamespace string`

**Review by YonkaFang — APPROVED** [review]

**Pull request merged**